### ISSUE TITLE
OpenStudio Geometry tab does not show my model

### ISSUE THREAD

**Issue by gagsters888** [NREL/OpenStudio#3968]
Issue overview
--------------

OpenStudio Geometry tab does not show my Building model 
## Current Behavior

I have drawn models using openstudio sketchup plugin and floorspace jps, however I am not able to see it under the Open Studio geometry 3-D view tab and I can only see a blank white page when I open the model's osm file. 

## Steps to Reproduce

1.
2.
3.
4.

### Environment

Some additional details about your environment for this issue (if relevant):
 - Platform (Operating system, version): Windows 10 64x
 - Version of OpenStudio (if using an intermediate build, include SHA):2.9.1

### Context

As I need to use it to do my project, I am unable to do it and my deadline is coming up soon, so I need to finish it soon. Please help me
![Geometry problemm](https://user-images.githubusercontent.com/64857050/81091161-68a47800-8f31-11ea-80ff-8f200b34c3c6.png)

**Comment by shorowit** [comment]
Any chance you have an NVIDIA graphics card? Does this help?

![image](https://user-images.githubusercontent.com/5861765/81092385-c0090080-8ebd-11ea-9e17-2489641c5687.png)

**Comment by gagsters888** [comment]
Hi shorowit,

I have changed my graphics to High-performance NVIDIA processor but I am still experience a blank white page in my openstudio geometry tab.

On a side note, sorry, I am a beginner to computer programming so I am not sure if I did the command prompt properly. But as you can see from the attachment, the code return me access denied.

Please advise me of the further steps on this matter. Thank you

![Command Prompt](https://user-images.githubusercontent.com/64857050/81136049-f82b4480-8f8c-11ea-92c7-9b88f6adc9b4.png).

**Comment by gagsters888** [comment]
Hi,

I have the same exact problem as the user johnsontimp as seen in the attachment that I have circled in red that I have attached in this file

My apologies, even after looking at the thread, I do not know how to solve this issue, thus I hope to seek your advice on this matter.

This is the url for the thread: https://github.com/NREL/OpenStudio/issues/2604
<img width="960" alt="Openstudio problem" src="https://user-images.githubusercontent.com/64857050/81171400-e8345480-8fce-11ea-9d80-b80ab2f309c7.png">

**Comment by shorowit** [comment]
You need to replace "%VERSION%" with the actual version you're running. And there are two lines you're supposed to run, not sure why you skipped the first. So:
```
set OPENSTUDIO_APPLICATION_DEBUG=1
c:\openstudio-2.9.1\bin\OpenStudioApp.exe > out.txt 2>&1
```
Then attach the out.txt file.

**Comment by gagsters888** [comment]
I am unable to access the openstudio command line interface from my openstudio 2.9.1 as it crashes everytime I open it from the openstudio folder.

P.S The previous command prompt that I have attached is the cmd in my computer, thus I am unable to code it.

**Comment by jmarrec** [comment]
Yes, open `cmd.exe` indeed, then type these two commands. This will launch the OpenStudioApplication while producing an out.txt file at your current location (by defaut that's your home folder when you open cmd.exe, as in `C:\Users\gagsters888` or whatever)

**Comment by gagsters888** [comment]
Ok, I have have attached the out.txt file. 
[out.txt](https://github.com/NREL/OpenStudio/files/4590388/out.txt)

**Comment by gagsters888** [comment]
Thx for the information.
This is the out. txt when I open my model in openstudio.
[out.txt](https://github.com/NREL/OpenStudio/files/4590406/out.txt)

**Comment by jmarrec** [comment]
@gagsters888 Can you record the out.txt until you navigate to the Geometry tab please?


-----

Note for us: There's something in the initial log that I haven't seen yet (but I don't use windows that much):

```
[openstudio.LocalLibraryController] <-2> Creating LocalLibraryController with base app 000000EF7C79F440
Warning: SetProcessDpiAwareness(2) failed: COM error 0xffffffff80070005  (Unknown error 0x0ffffffff80070005), using 0 ((null):0, (null))
Warning: QWindowsWindow::setGeometry: Unable to set geometry 5x13+640+280 on QWidgetWindow/'QLabelClassWindow'. Resulting geometry:  120x13+640+280 (frame: 8, 31, 8, 8, custom margin: 0, 0, 0, 0, minimum size: 0x0, maximum size: 16777215x16777215). ((null):0, (null))
Warning: QWindowsWindow::setGeometry: Unable to set geometry 5x13+640+280 on QWidgetWindow/'QLabelClassWindow'. Resulting geometry:  120x13+640+280 (frame: 8, 31, 8, 8, custom margin: 0, 0, 0, 0, minimum size: 0x0, maximum size: 16777215x16777215). ((null):0, (null))
Warning: QWindowsWindow::setGeometry: Unable to set geometry 5x13+640+280 on QWidgetWindow/'QLabelClassWindow'. Resulting geometry:  120x13+640+280 (frame: 8, 31, 8, 8, custom margin: 0, 0, 0, 0, minimum size: 0x0, maximum size: 16777215x16777215). ((null):0, (null))
Warning: QWindowsWindow::setGeometry: Unable to set geometry 5x13+640+280 on QWidgetWindow/'QLabelClassWindow'. Resulting geometry:  120x13+640+280 (frame: 8, 31, 8, 8, custom margin: 0, 0, 0, 0, minimum size: 0x0, maximum size: 16777215x16777215). ((null):0, (null))
```

**Comment by jmarrec** [comment]
Could you send your model to me actually? If you don't want to share it publicly you can send it via email to julien at effibem dot com. I'll try to see if I can see it in the Geometry Tab, and if I can then we'll have to troubleshoot more with you, if I also can't see it I can investigate myself.

**Comment by gagsters888** [comment]
Hi, 

Thank you so much for your help.

I managed to see my model in the geometry tab now.

Apparently I could not see the model when I am extending my screen to an external monitor but now when I dont extend my screen to an external monitor, I managed to see my model when I am just using my laptop screen.

In addition to that, I think the changing of the 3-D settings in my nvidia graphic card helps as well.

Thank you so much for all the help on this matter. Really appreciate them!
